pkg/daemon/ceph/osd/volume.go

```diff
@@ -50,16 +50,18 @@ const (
 	cephVolumeCmd        = "ceph-volume"
 	cephVolumeMinDBSize  = 1024 // 1GB
 	lvmFilter            = `filter = [ "a|^/mnt/.*|", "r|.*|" ]`
+	lvmFilterPVBacked    = `filter = [ "a|^/mnt/.*|", "a|^/dev/.*|", "r|.*|" ]`
```

Comment:
In the PVC case, we must skip `/dev/loop` devices, so please change `"a|^/dev/.*|"` to accept everything accept loop devices. Thanks.

Reply:
Thanks, changed as you said.